AllReadyApp/Web-App/AllReady/web.config

```diff
@@ -4,6 +4,10 @@
     <handlers>
       <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified" />
     </handlers>
-    <aspNetCore processPath=".\AllReady.exe" arguments="" forwardWindowsAuthToken="false" stdoutLogEnabled="false" />
+    <aspNetCore processPath="%LAUNCHER_PATH%" arguments="%LAUNCHER_ARGS%" forwardWindowsAuthToken="false" stdoutLogEnabled="false" />
```

Comment:
Please remove this change until we get our act together w.r.t. deploying on Azure with environment variables.